firmware/shared/device/mcp/mcp4728.hpp

```diff
@@ -74,13 +80,33 @@ class MCP4728 : public DigitalAnalogConverter<kMcp4728NumChannels> {
         LoadVoltages();
     }
 
+    void Init()
+    {
+        SetVref(VREF::VDD, VREF::VDD, VREF::VDD, VREF::VDD);
+    }
+
+    void SetVref(VREF Ch_A, VREF Ch_B, VREF Ch_C, VREF Ch_D){
```

Comment:
this should just be in the constructor tbh. by the time its used at app level this wont even be exposed.

Reply:
please also use google style guide (but delete this function, just as an example)
```suggestion
    void SetChannelVoltageReferences(VoltageRef chan_a, VoltageRef chan_b,, VoltageRef chan_c, VoltageRef chan_d){
```

Reply:
I'll make the function private and call it in the constructor, but the function should still exist imo